---
layout: default
---
## Publications

<sup>\*</sup> denotes co-first authors

### Preprints

O'Connor C, **Keele GR**, Martin W, Stodola T, Gatti D, Hoffman BR, Konstanje R, Churchill GA, Reinholdt LG. Cell morphology QTL reveal gene by environment interactions in a genetically diverse cell population. *bioRxiv*. [https://doi.org/10.1101/2023.11.18.567597](https://doi.org/10.1101/2023.11.18.567597)

Nemkov T, Stephenson D, Earley EJ, **Keele GR**, Hay A, Erickson C, Dzieciatkowska M, Key AM, Moore A, Stone M, Deng X, Kleinman S, Spitalnik SL, Hod EA, Hansen K, Churchill GA, Roubinian N, Norris PJ, Busch MP, Zimring JC, Page GP, D'Alessandro A. Biological and genetic determinants of red blood cell glycolysis. *bioRxiv*. [https://doi.org/10.1101/2023.09.11.557250](https://doi.org/10.1101/2023.09.11.557250)

**Keele GR**, Maurizio PL, Oreper D, Valdar W. Bayesian decision theoretic design of two-founder experimental crosses given diallel data. *bioRxiv*. [https://doi.org/10.1101/489682](https://doi.org/10.1101/489682)

### Peer-reviewed (first author)

**Keele GR**, Zhang J, Szpyt J, Korstanje R, Gygi SP, Churchill GA, Schweppe DK. Global and tissue-specific aging effects on murine proteomes. *Cell Reports*. 2023 July 4. [https://doi.org/10.1016/j.celrep.2023.112715](https://doi.org/10.1016/j.celrep.2023.112715)

Zhang T<sup>\*</sup>, **Keele GR**<sup>\*</sup>,Gerdes Gyuricza I, Vincent M, Brunton C, Bell TA, Hock P, Shaw GD, Munger SC, de Villena FP-M, Ferris MT, Paulo JA, Gygi SP, Churchill GA. Multi-omics analysis identifies drivers of protein phosphorylation. *Genome Biology*. 2023 Mar 21. [https://doi.org/10.1186/s13059-023-02892-2](https://doi.org/10.1186/s13059-023-02892-2)

**Keele GR**. Which mouse multiparental population is right for your study? The Collaborative Cross inbred strains, their F1 hybrids, or the Diversity Outbred population. *G3*. 2023 Feb 3. [https://doi.org/10.1093/g3journal/jkad027](https://doi.org/10.1093/g3journal/jkad027)

Crouse WL<sup>\*</sup>, **Keele GR**<sup>\*</sup>, Gastonguay MS, Churchill GA, Valdar W. A Bayesian model selection approach to mediation analysis. *PLoS Genetics*. 2022 May 9. [https://doi.org/10.1371/journal.pgen.1010184](https://doi.org/10.1371/journal.pgen.1010184)

**Keele GR**<sup>\*</sup>, Zhang T<sup>\*</sup>, Pham DT, Vincent M, Bell TA, Hock P, Shaw GD, Paulo JA, Munger SC, de Villena FP-M, Ferris MT, Gygi SP, Churchill GA. Regulation of protein abundance in genetically diverse mouse populations. *Cell Genomics*. 2021 Aug 30. [https://doi.org/10.1016/j.xgen.2021.100003](https://doi.org/10.1016/j.xgen.2021.100003)

**Keele GR**<sup>\*</sup>, Prokop JW<sup>\*</sup>, Hong H, Holl K, Littrell J, Deal AW, Kim Y, Kyle PB, Attipoe E, Johnson AC, Uhl KL, Sirpilla OL, Jahanbakhsh S, Levy S, Valdar W, Garrett MR, Solberg Woods LC. *Sept8/SEPTIN8* involvement in cellular structure and kidney damage is identified by genetic mapping and a novel human tubule hypoxic model. *Scientific Reports*. 2021 Jan 22. [https://doi.org/10.1038/s41598-021-81550-8](https://doi.org/10.1038/s41598-021-81550-8) 

Lerner AM<sup>\*</sup>, Hepperla AJ<sup>\*</sup>, **Keele GR**<sup>\*</sup>, Meriesh H, Yumerefendi H, Restrepo D, Zimmerman S, Bear J, Kuhlman B, Davis IJ, Strahl BD. An optogenetic switch for the Set2 methyltransferase provides evidence for transcription-dependent and -independent dynamics of H3K36 methylation. *Genome Research*. 2020 Oct 5. [https://doi.org/10.1101/gr.264283.120](https://doi.org/10.1101/gr.264283.120)

**Keele GR**<sup>\*</sup>, Quach BC<sup>\*</sup>, Israel JW, Chappell GA, Lewis L, Safi A, Simon JM, Cotney P, Crawford GE, Valdar W, Rusyn I, Furey TS. Integrative QTL analysis of gene expression and chromatin accessibility identifies multi-tissue patterns of genetic regulation. *PLoS Genetics*. 2020 Jan 21. [https://doi.org/10.1371/journal.pgen.1008537](https://doi.org/10.1371/journal.pgen.1008537) 

**Keele GR**<sup>\*</sup>, Crouse WL<sup>\*</sup>, Kelada SNP, Valdar W. Determinants of QTL mapping power in the realized Collaborative Cross. *G3*. 2019 May 1. [https://doi.org/10.1534/g3.119.400194](https://doi.org/10.1534/g3.119.400194)

**Keele GR**, Prokop JW, He H, Holl K, Littrell J, Deal A, Francic S, Cui L, Gatti DM, Broman KW, Tschannen M, Shirng-Wern T, Zagloul M, Kim Y, Baur B, Fox J, Robinson M, Levy S, Flister MJ, Mott R, Valdar W, Solberg Woods LC. Genetic fine-mapping and identification of candidate genes and variants for adiposity traits in outbred rats. *Obesity*. 2017 Nov 28. [https://doi.org/10.1002/oby.22075](https://doi.org/10.1002/oby.22075)

### Peer-reviewed (middle author)

Nemkov T, Key A, Stephenson D, Earley EJ, **Keele GR**, Hay AM, Amireault P, Casimir MD, Dzieciatkowska M, Reisz JA, Deng X, Stone M, Kleinman SH, Spitalnik SL, Hansen KC, Norris PJ, Churchill GA, Busch MP, Roubinian NH, Page GP, Zimring JC, Arduini A, A D’Alessandro. Genetic regulation of carnitine metabolism controls lipid damage repair and RBC hemolysis in vivo and in vitro. *Blood*. 2024 Mar 21. [https://doi.org/10.1182/blood.2024023983](https://doi.org/10.1182/blood.2024023983)

Aygün N, Liang D, Crouse WL, **Keele GR**, Love MI, Stein JL. Inferring cell-type-specific causal gene regulatory networks during human neurogenesis. *Genome Biology*. 2023 May 30. [https://doi.org/10.1186/s13059-023-02959-0](https://doi.org/10.1186/s13059-023-02959-0)

Aydin S, Pham DT, Zhang T, **Keele GR**, Skelly DA, Paulo JA, Pankratz M, Choi T, Gygi SP, Reinholdt LG, Baker CL, Churchill GA, Munger SC. Genetic dissection of the pluripotent proteome through multi-omics data integration. *Cell Genomics*. 2023 Mar 23. [https://doi.org/10.1016/j.xgen.2023.100283](https://doi.org/10.1016/j.xgen.2023.100283)

Gastonguay MS, **Keele GR**, Churchill GA. The Trouble with Triples: Examining the Impact of Measurement Error in Mediation Analysis. *Genetics*. 2023 Mar 18. [https://doi.org/10.1093/genetics/iyad045](https://doi.org/10.1093/genetics/iyad045)

Zhang Q<sup>\*</sup>, Linke V<sup>\*</sup>, Overmyer KA, Traeger LL, Kashara K, Miller IJ, Manson DE, Polaske T, Kerby RL, Kemis JH, Trujillo EA, Reddy TR, Russell JD, Schueler KL, Stapleton DS, Rabaglia ME, Seldin M, Gatti DM, **Keele GR**, Pham DT, Gerdt JP, Vivas EI, Lusis AJ, Keller MP, Churchill GA, Blackwell HE, Broman KW, Attie AD, Coon JJ, Rey FE. Genetic mapping of microbial and host traits reveals production of immunomodulatory lipids by *Akkermansia muciniphila* in the murine gut. *Nature Microbiology*. 2023 Feb 9. [https://doi.org/10.1038/s41564-023-01326-w](https://doi.org/10.1038/s41564-023-01326-w)

Yu Q, Liu X, Keller MP, Navarrete-Perea J, Zhang T, Fu S, Vaites L, Shuken S, Schmid E, **Keele GR**, Li J, Huttlin E, Rashan E, Simcox J, Churchill GA, Schweppe DK, Attie AD, Paulo JA, SP Gygi. Sample multiplexing-based targeted pathway proteomics with real-time analytics reveals the impact of genetic variation on protein expression. *Nature Communications*. 2023 Feb 2. [https://doi.org/10.1038/s41467-023-36269-7](https://doi.org/10.1038/s41467-023-36269-7)

Le TH, Crouse WL, **Keele GR**, Holl K, Seshie O, Tschannen M, Craddock A, Das SK, Szalanczy AM, McDonald B, Grzybowski M, Sharma NK, Geurts AM, Key CC, Hawkins G, Valdar W, Mott R, Solberg Woods LC. Genetic mapping of multiple metabolic traits identifies novel genes for adiposity, lipids and insulin secretory capacity in outbred rats. *Diabetes*. 2022 Oct 11. [https://doi.org/10.2337/db22-0252](https://doi.org/10.2337/db22-0252)

Vincent M<sup>\*</sup>, Gerdes Gyuricza I<sup>\*</sup>, **Keele GR**, Gatti DM, Keller MP, Broman KW, Churchill GA. QTLViewer: An interactive web tool for genetic analysis in the Collaborative Cross and Diversity Outbred mouse populations. *G3*. 2022 June 15. [https://doi.org/10.1093/g3journal/jkac146](https://doi.org/10.1093/g3journal/jkac146)

Crouse WL<sup>\*</sup>, Das SK<sup>\*</sup>, Le TH, **Keele GR**, Holl K, Seshie O, Craddock L, Sharma N, Comeau M, Langefeld CD, Hawkins G, Mott R, Valdar W, Solberg Woods LC. Transcriptome-wide analyses of adipose tissue in outbred rats reveal novel genetic regulatory mechanisms relevant to obesity for humans. *Physiological Genomics*. 2022 June 1. [https://doi.org/10.1152/physiolgenomics.00172.2021](https://doi.org/10.1152/physiolgenomics.00172.2021)

Gerdes Gyuricza I, Chick JM, **Keele GR**, Deighan AG, Munger SC, Korstanje RC, Gygi SP, Churchill GA. Genome-wide transcript and protein analysis highlights the role of protein homeostasis in the aging mouse heart. *Genome Research*. 2022 March 11. [https://doi.org/10.1101/gr.275672.121](https://doi.org/10.1101/gr.275672.121)

Čuklina J, Lee CH, Williams EG, Sajic T, Collins BC, Rodríguez Martínez M, Sharma VS, Wendt F, Goetze S, **Keele GR**, Wollscheid B, Aebersold R, Pedrioli PGA. Diagnostics and correction of batch effects in large-scale proteomics studies: a tutorial. *Molecular Systems Biology*. 2021 Aug 1. [http://doi.org/10.15252/msb.202110240](http://doi.org/10.15252/msb.202110240)

Zhang T, **Keele GR**, Churchill GA, Gygi SP, Paulo JA. Strain-specific peptide (SSP) interference reference sample: A genetically encoded quality control standard for isobaric tagging strategies. *Analytical Chemistry* 2021 Mar 18. [https://doi.org/10.1021/acs.analchem.0c05483](https://doi.org/10.1021/acs.analchem.0c05483)

Hsiao K, Noble C, Pitman W, Yadav N, Kumar S, **Keele GR**, Terceros A, Kanke M, Conniff T, Cheleuitte-Nieves C, Tolwani R, Sethupathy P, Rajasethupathy P. A Thalamic Orphan Receptor Drives Variability in Short-Term Memory. *Cell*. 2020 Sep 29. [https://doi.org/10.1016/j.cell.2020.09.011](https://doi.org/10.1016/j.cell.2020.09.011)

Linke V, Overmyer KA, Miller IJ, Brademan DR, Hutchins PD, Trujillo EA, Reddy TR, Russell JD, Cushing EM, Schueler KL, Stapleton DS, Rabaglia ME, Keller MP, Gatti DM, **Keele GR**, Pham D, Broman KW, Churchill GA, Attie AD, Coon JJ. A large-scale genome-lipid association map guides lipid identification. *Nature Metabolism*. 2020 Sep 21. [https://doi.org/10.1038/s42255-020-00278-3](https://doi.org/10.1038/s42255-020-00278-3)

Skelly DA, Czechanski A, Byers C, Aydin S, Spruce C, Olivier C, Choi K, Gatti DM, Raghupathy N, **Keele GR**, Stanton A, Vincent M, Dion S, Greenstein I, Pankratz M, Porter DK, Martin W, O’Connor C, Qin W, Harrill AH, Choi T, Churchill GA, Munger SC, Baker CL, Reinholdt LG. Mapping the Effects of Genetic Variation on Chromatin State and Gene Expression Reveals Loci That Control Ground State Pluripotency. *Cell Stem Cell*. 2020 Aug 13. [https://doi.org/10.1016/j.stem.2020.07.005](https://doi.org/10.1016/j.stem.2020.07.005)

Maurizio PL, Ferris MT, **Keele GR**, Miller DR, Shaw GD, Whitmore AC, West A, Morrison CR, Noll KE, Plante KS, Cockrell AS, Threadgill DW, Pardo-Manuel de Villena F, Baric RS, Heise MT, Valdar W. Bayesian diallel analysis reveals *Mx1*-dependent and *Mx1*-independent effects on response to influenza A virus in mice. *G3*. 2017 Nov 29. [https://doi.org/10.1534/g3.117.300438](https://doi.org/10.1534/g3.117.300438)

Mosedale M<sup>\*</sup>, Kim Y<sup>\*</sup>, Brock WJ, Roth SE, Wiltshire T, Eaddy JS, **Keele GR**, Corty RW, Xie Y, Valdar W, Watkins PB. Candidate risk factors and mechanisms for Tolvaptan-induced liver injury are identified using a Collaborative Cross approach. *Toxicological Sciences*. 2017 Apr 1. [https://doi.org/10.1093/toxsci/kfw269](https://doi.org/10.1093/toxsci/kfw269)

Lange EM, Ribado JV, Zuhlke KA, Johnson AM, **Keele GR**, Li J, Wang Y, Duan Q, Li G, Gao Z, Li Y, Xu J, Zheng SL, Cooney KA. Assessing the cumulative contribution of new and established common genetic risk factors to early-onset prostate cancer. *Cancer Epidemiol Biomarkers Prev*. 2016 May. [https://doi.org/10.1158/1055-9965.EPI-14-0995](https://doi.org/10.1158/1055-9965.EPI-14-0995)

Pathak A, Wenzlaff AS, Hyland PL, Cote ML, **Keele GR**, Land S, Boulton ML, Schwartz AG. Apoptosis-related single nucleotide polymorphisms and the risk of non-small cell lung cancer in women. *J Cancer Ther Res*. 2014 Jan 31. [https://doi.org/10.7243/2049-7962-3-1](https://doi.org/10.7243/2049-7962-3-1)

Lange EM, Johnson AM, Wang Y, Zuhlke KA, Lu Y, Ribado JV, **Keele GR**, Li J, Duan Q, Li G, Gao Z, Li Y, Xu J, Isaacs WB, Zheng S, Cooney KA. Genome-wide association scan for variants associated with early-onset prosate cancer. *PLoS One*. 2014 Apr 16. [https://doi.org/10.1371/journal.pone.0093436](https://doi.org/10.1371/journal.pone.0093436)

Beebe-Dimmer JL, Colt JS, Ruterbusch JJ, **Keele GR**, Purdue MP, Wacholder S, Graubard BI, Davis F, Chow WH, Schwartz KL. Body mass index and renal cell cancer: the influence of race and sex. *Epidemiology*. 2012 Nov. [https://doi.org/10.1097/EDE.0b013e31826b7fe9](https://doi.org/10.1097/EDE.0b013e31826b7fe9)

Asmar R, Beebe-Dimmer JL, Korgavkar K, **Keele GR**, Cooney KA. Hypertension, obesity and prostate cancer biochemical recurrence after radical prostatectomy. *Prostate Cancer Prostatic Dis*. 2012 Aug 21. [https://doi.org/10.1038/pcan.2012.32](https://doi.org/10.1038/pcan.2012.32)

Beebe-Dimmer JL, Iyer PT, Nriagu JO, **Keele GR**, Mehta S, Meliker JR, Lange EM, Schwartz AG, Zuhlke KA, Schottenfeld D, Cooney KA. Genetic variation in glutathione S-transferase omega-1, arsenic methyltransferase and methylene-tetrahydrofolate reductase, arsenic exposure and bladder cancer: a case-control study. *Environ Health*. 2012 Jun 29. [https://doi.org/10.1186/1476-069X-11-43](https://doi.org/10.1186/1476-069X-11-43)

[back](./)
